project-reports/2023/2023-Q3-Hyperledger-Aries.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: default
+title: 2023 Q3 Hyperledger Aries
+parent: 2023
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+nav_exclude: true
+---
+
+# Project Health
+
+Hyperledger Aries is healthy, seeing lots of activity overall, especially in all
+of the Aries communities, including [Aries Framework JavaScript], [Aries
+Bifold] (mobile wallet), [Aries Cloud Agent Python], [Aries VCX] and [Aries
+Framework Go]. Lots of progress being made on all of those efforts!
+
+[Aries Framework JavaScript]: https://github.com/hyperledger/aries-framework-javascript
+[Aries Bifold]: https://github.com/hyperledger/aries-mobile-agent-react-native
+[Aries Cloud Agent Python]: https://github.com/hyperledger/aries-cloudagent-python
+[Aries VCX]: https://github.com/hyperledger/aries-vcx
+[Aries Framework Go]: https://github.com/hyperledger/aries-framework-go
+
+# Questions/Issues for the TOC
+
+None.
+
+# Releases
+
+The following Aries releases occurred in the last quarter:
+
+-   [Aries Cloud Agent Python] -- 0.8.1, 0.8.2, 0.9.0
+-   [Aries Framework JavaScript] -- v0.4.0, v0.4.1alpha.1-20
+-   [Aries Askar] -- v0.2.9
+-   [Aries VCX Releases] -- v0.54.0-v0.57.1
+-   [Aries Framework Go] -- v0.2.0, v0.3.0, v0.3.1,v0.3.2
+-   [Aries Bifold] -- v0.0.0 - v0.1.0-alpha.53
+
+Interoperability status can be seen
+here: [https://aries-interop.info](https://aries-interop.info).
+
+# Overall Activity in the Past Quarter
+
+Per the [Aries Activity Dashboard] for the months of April-June 2023, Aries
+codebases had 387 PRs (down 12%) from 70 contributors (down 13%).
+
+[Aries Activity Dashboard]: https://insights.lfx.linuxfoundation.org/projects/hyperledger%2Faries/dashboard;subTab=technical?time=%7B%22from%22:%222023-04-01T07:00:00.000Z%22,%22type%22:%22absolute%22,%22to%22:%222023-06-30T07:00:00.000Z%22%7D
+
+* There was a lot of activity towards integrating [Hyperledger
+  AnonCreds](https://github.com/hyperledger/anoncreds-rs) into all of the Aries
+  Frameworks to enable ledger-agnostic implementations, with the [Aries
+  Framework JavaScript] 0.4.0 as the biggest milestone. [Aries Framework
+  JavaScript] 0.4.0 includes Aries Askar and the shared components, including
+  [Hyperledger AnonCreds], fully integrated, and initial support for
+  [OpenID4VCs] protocols.
+    * [Aries Bifold] using AFJ 0.4.0 was deployed with ease, including updates
+      to existing Mobile Wallets being updated on the fly from using the Indy
+      SDK to using Askar.
+    * Instances of [Aries Framework JavaScript] using non-Indy ledgers were
```

Comment:
"deployed" was used in the two bullets above, can you clarify what that means? "deployed by adopters that have upgraded existing solutions to the new version"? or something else? if the former, can you add some context (doesn't need the details if the adopter's solution isn't a public reference) of the deployed solution? thanks!

Reply:
Updated the wording. Use — deployed by adopters that have upgraded their implementations. The smooth transition from Indy SDK storage to Askar is significant - requiring the safe updating of stored data from one format to another.